cloudinit/sources/DataSourceAzure.py

```diff
@@ -2140,24 +2145,6 @@ def maybe_remove_ubuntu_network_config_scripts(paths=None):
                 util.del_file(path)
 
 
-def _is_platform_viable(seed_dir):
-    """Check platform environment to report if this datasource may run."""
-    with events.ReportEventStack(
```

Comment:
Just for our understanding Is there a motivation in general to reduce event frame logging in Azure during early boot to avoid excessive or noisy eventing? I just want to know if we have general guidance of what type of events Azure would like to see vs not.

Reply:
Added @azure_ds_telemetry_reporter to is_platform_viable.